docs/current/guides/757394-use-service-containers.md

```diff
@@ -12,6 +12,9 @@ import TabItem from "@theme/TabItem";
 import Embed from '@site/src/components/atoms/embed.js'
 
 # Use Service Containers in Dagger
+:::warning
+Dagger v0.8.8 includes a breaking change for binding service containers to containers. You must call Container.service to get a service instead of binding directly to a container. See [service containers to containers](#container-services-to-containers) for examples.
```

Comment:
```suggestion
Dagger v0.8.8 includes a breaking change for binding service containers to containers. You must call `Container.service()` to get a service instead of binding directly to a container. See [service containers to containers](#container-services-to-containers) for examples.
```